data/korath/korath missions.txt

```diff
@@ -257,3 +257,179 @@ mission "Small Scale Delivery"
+				to display
+					"reputation: Hai (Unfettered)" < 30
+			`	Batu looks less satisfied. "We've found far less than we expected, and there aren't any live Korath - or even Korath bodies. Our investigators suspect we've found a small part of a larger operation to pillage and sell Korath outfits. The operation's Stormhold facilities were just distribution centers for sales and staging areas for transfers of Korath. I hope shutting down this part of the operation will end it for good, but that seems overly optimistic."`
+
+			# This seemingly abrupt question is to ensure the player notices the word "Korath."
+			choice
+				`	"Korath? Who are the Korath?"`
+				`	"Why are there so many Korath artifacts?"`
+					goto somany
+			`	Choot'k responds. "The Korath are group of lizards that lived south of Hai space until few centuries ago. They used to be obsessed with art, music, and other esoterics, until they got desperate for resources and invaded us. Their technology is far beyond ours, and they used it to nearly wipe themselves out in bloody infighting. We learned a lot from them: even the most aggressive Unfettered are wary of attacking Hai planets, lest our species destroy itself like the Korath did. In all our raids in the Wah Ki system, never once have the Unfettered targeted civilian areas on Cloudfire." Batu appears shocked by this.`
+			label somany
+			`	Choot'k continues. "The most advanced group of surviving Korath lives in systems near yours, according to pirate data. It seems the Korath have been raiding ships in pirate and Syndicate systems for food, while the pirates have been disabling and pillaging Korath ships for tech. I'm not sure why the Korath have limited themselves to piracy. They certainly have the firepower to dominate a few dozen of your planets.`
+			`	"From our race's records, it is quite unlike the Korath to be so close to a more primitive race without making demands. Their resources may be committed to another war, or they may be building up supplies for a shock and awe style attack. It's anyone's guess at this point.`
+			`	"Even the Unfettered are concerned about this. A few dozen human planets under the dominion of the Korath could make them a force to be reckoned with again. They could even be a threat to the Hai."`
+			choice
+				`	"Sounds foreboding."`
+					goto foreboding
+				`	"When do I get my cut of these spoils?"`
+			`	"Ha! I'd think you were a True Hai if it weren't for the missing fur. Don't worry, captain, I'll make sure the others give you a suitable payment."`
+			`	Batu sighs as Choot'k continues. "We've made an unpleasant discovery recently. The Korath have stealth drones hiding in two pirate systems: Alcyone and Almach. They've been using these to spy on nearby traffic, including us. It's also... eh, Batu?"`
```

Comment:
stealth drones? I dont think they're really stealthed?

Reply:
They're stealthy enough so tier 1 races couldn't find them. It may not be part of the standard design, but these are stealthy.

Reply:
yeah I know I just think we shouldnt use that word here but another one, idk, discrete, camouflaged, hidden,... but not stealthed because that has a specific meaning in the game

Reply:
The game is not presenting the word "stealth" as a fact; rather, it is the interpretation of the human and Hai investigators, which their leaders are parroting as the absolute truth. If you can't find drones for years, and they've been designed specifically, so you can't find them, then they're stealth drones from your point of view. Part of the player's job is to separate the real truth from the speaker's interpretation of the truth.

Reply:
I'm not sure "stealth" really has that much of a specific meaning in-game, does it? It doesn't really have much meaning for me, at least. We have visible ships, and cloaked ships. So stealth is pretty much just a lore thing, not something that is in use in-game.

I'm good with these drones being called "stealth"